sdk/textanalytics/azure-ai-textanalytics/tests/test_multiapi_async.py

```diff
@@ -27,6 +27,6 @@ def test_v3_0_api_version(self, resource_group, location, text_analytics_account
     @GlobalTextAnalyticsAccountPreparer()
     def test_v3_1_preview_1_api_version(self, resource_group, location, text_analytics_account, text_analytics_account_key):
         credential = AzureKeyCredential(text_analytics_account_key)
-        client = TextAnalyticsClient(text_analytics_account, credential, api_version=ApiVersion.V3_1_preview_1)
+        client = TextAnalyticsClient(text_analytics_account, credential, api_version=ApiVersion.V3_1_PREVIEW_1)
```

Comment:
should we also make a call to verify? or do we test that somewhere else...

Reply:
I think this is an adequate test. The generated client's base url contains v3.0 or v3.1-preview.1 depending on the , and we use the base url as the base for all of our function calls